sphinx/conf.py

```diff
@@ -6,11 +6,13 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
+import os
+
 project = "skore"
 copyright = "2024, Probabl"
 author = "Probabl"
-# version = "0"
-# release = "0"
+version = os.environ["SPHINX_VERSION"]
+release = os.environ["SPHINX_VERSION"]
```

Comment:
If version is always equal to release variable why not keep only one of them?

Reply:
Changed: release is now set with the tag.